package.json

```diff
@@ -7,12 +7,25 @@
   "author": "Akshay S Dinesh <asdofindia@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "jspm bundle src/ascii-unicode-converter.js dist/bundle.js --skip-source-maps"
+    "run-server": "node_modules/http-server/bin/http-server --cors -p 9000",
+    "postinstall": "cp node_modules/siimple/dist/*.min.css src/assets/ && cp node_modules/siimple-colors/dist/*.min.css src/assets/",
+    "build:css": "cp -R src/assets/ dist/",
```

Comment:
Makes sense. I'll add more commits that fixes it directly using #2. Like I said, my sole interest was minimizing the surface of build. Going this way sort of forces you into a workflow that is reliant on the build as well.